config/packages/nelmio_security.yaml

```diff
@@ -22,3 +22,13 @@ nelmio_security:
     enabled: true
     hsts_max_age: 31536000 # 365 days
     hsts_subdomains: true
+
+when@dev: &webpack_dev
```

Comment:
Nice trick! But may I suggest renaming `webpack` to `nelmio` or something more relevant?